packages/dev-server/package.json

```diff
@@ -65,7 +65,7 @@
     "command-line-usage": "^7.0.1",
     "debounce": "^1.2.0",
     "deepmerge": "^4.2.2",
-    "ip": "^2.0.1",
+    "internal-ip": "^6.2.0",
```

Comment:
had to use 6.2.0 since it's a CommonJS module which works with our current CommonJS output
as soon as we go to ESM for our modules, we can try 7.0.0